.github/workflows/build.yml

```diff
@@ -190,7 +191,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           MERGE_METHOD: squash
-          MERGE_LABELS: ""
+          MERGE_LABELS: "automerge,!hold"
+          MERGE_REMOVE_LABELS: "automerge"
```

Comment:
IMHO it would be simple not to reach this step at all, i.e. not approve PR and fail the job. How about to have a step before "Approve PR" like the following:

```
- if: ${{ steps.verify_pr.conclusion == 'failure }}
  run: exit 1
```

With that we do no need "Check for automerge" step,  nor any condition on "Merge PR" step.

Reply:
I did this change but seems to not working in the case of conflict. So, I am going to revert this change unless I missed something. 
Here is the test PR: https://github.com/openshift-helm-charts/sandbox/pull/7

Reply:
It is because `verify_pr` step never fails - the last command executed there is `cd ..` and it is always successful.

Reply:
There are scenarios where `verify_pr` fails. The Python script calls `sys.exit(1)` whenever there is an issue with checks.
In the above [test PR](https://github.com/openshift-helm-charts/sandbox/pull/7) all checks were passing.  But the merge didn't happen because there is a conflict.  So, we cannot depend on the failure of `verify_pr` step to decide to make the chart release.

Reply:
> There are scenarios where verify_pr fails. The Python script calls sys.exit(1) whenever there is an issue with checks.

My bad, the script is really configured to fail fast, I missed that.

However,  the automerge action does not fail if merge cannot be done due to conflicts. A way to check this would be to check the PR status via something like this:

```
var pr = github.pulls.get({
      owner: context.repo.owner,
      repo: context.repo.repo,
      pull_number: ${{ github.event.pull_request.number }}
     });
if (!pr.merged) {
       core.setFailed('not merged')
}
```

And this should happen probably in some sort of loop. Checking for removed label is just not enough - there could be still a room for failing the merge.

Reply:
Done. df8a9de9cae73b7d95e2477398200d5e20e23278